client/landing/gutenboarding/onboarding-block/signup-form/index.tsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+// /**
+//  * Internal dependencies
+//  */
+import {
+	usePasswordlessSignUp,
+	UsePasswordlessSignUpStatus,
+	Provider,
+	Client,
+} from '@automattic/authentication';
+import { Button, TextControl, Modal } from '@wordpress/components';
+import FormLabel from 'components/forms/form-label';
+import { __ as NO__ } from '@wordpress/i18n';
+import config from '../../../../config';
+import './style.scss';
+
+const SignupForm = () => {
+	const [ emailVal, setEmailVal ] = useState( '' );
+	const { signUp, status, error } = usePasswordlessSignUp();
+
+	const handleSignUp = ( event: React.FormEvent< HTMLFormElement > ) => {
+		event.preventDefault();
+		signUp( emailVal );
```

Comment:
Once the work in https://github.com/Automattic/wp-calypso/pull/38823/files#diff-5fdbca26e3ba6bb6ea1026beb3049c5fR76 is ready, we can swap out the method and this will be golden.